### PR TITLE
Evaluating the Search Phase of Neural Architecture Search

### PR DESCRIPTION
**Pull request by AntoineTheb**

### CODE REVIEW

File: article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+layout: review
+title: "Evaluating the Search Phase of Neural Architecture Search"
+tags: NAS deep-learning
```
Comment (jhlegarreta) [inline article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md]: Does not seem like NAS is listed in the tags file.

File: article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- For all of the 10 runs, the Kendall Tau metrics are close to zero, which suggests a lack of correlation between the WS rankings and the true one.
+
+# Conclusions
+
+> We have observed that, surprisingly, the search policies of state-of-the-art NAS techniques are no better than random, and have traced the reason for this to the use of (i) a constrained search space and (ii) weight sharing, which shuffles the architecture ranking during the search, thus negatively impacting it.
+
+# Remarks
+
+
+# References
```
Comment (jhlegarreta) [inline article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md]: I would remove the sections if they are void of contents.

Comment (AntoineTheb) [reply article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md]: Woops indeed ! Fixed in 540cfa0

File: article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md (hunk 1 above)
Comment (AntoineTheb) [reply article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md]: Argh I forgot to include it! Fixed in 540cfa0

**Pull request merged**